gnovm/cmd/gno/bug_test.go

```diff
@@ -0,0 +1,17 @@
+package main
```

Comment:
Please add more unit tests for this gno bug functionality, I don't think checking that the help command works is a valid smoke test 🙏